daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -248,7 +250,7 @@ runStart
                   void $ waitAnyCancel =<< mapM (async . waitExitCode) [navigatorPh,sandboxPh,jsonApiPh]
 
     where
-        navigatorPort = NavigatorPort 7500
+        navigatorPort = fromMaybe (NavigatorPort 7500) navigatorPortM
```

Comment:
Is there a reason why we do the defaulting here rather than in `optparse-applicative`? I guess the same applies to the sandbox port as well which is probably my doing but I don’t remember why we handled it this way.

Reply:
I don't think there's a good reason. I'll move this default over.